### PR TITLE
load jagex fonts

### PR DESCRIPTION
**Pull request by conker-rsc**
Loads original .jf files at startup for better font rendering, especially on macOS.

Added option toggle such that users may choose to use system fonts if desired.

### CODE REVIEW

**Review by Hubcapp — APPROVED** [review]
I made a few changes so that loading jagex font or system font can be done without reloading the client. :-) I think this is a great PR. Thanks for doing the research and hard work on this.

File: src/Game/Client.java [hunk 1]
```diff
@@ -1855,6 +1855,32 @@ private static String processClientCommand(String line) {
...
+              displayMessage("Unable to load system fonts...", CHAT_QUEST);
+            }
+          }
+          break;
+        case "jffonts":
+        {
+          GameApplet.loadJfFonts();
+          displayMessage("Loaded jf fonts", CHAT_QUEST);
+          break;
+        }
```
Comment (conker-rsc) [inline src/Game/Client.java:1872]: Additionally, these are separate from the options toggles, so they get out of sync with those settings if you invoke them... Does it matter here?

File: src/Game/Client.java [hunk 2]
```diff
@@ -1855,6 +1855,32 @@ private static String processClientCommand(String line) {
...
+        case "shellstring": {
```
Comment (conker-rsc) [inline src/Game/Client.java:1873]: THANK YOU FOR THIS!!!

**Comment by Hubcapp** [comment]
Tested on Windows, Macintosh, and Linux. LGTM!

**Pull request merged**